src/plugins/intel_cpu/src/transformations/snippets/aarch64/pass/snippets_mark_skipped.cpp

```diff
@@ -0,0 +1,261 @@
+
+
+namespace ov {
+namespace intel_cpu {
+
+namespace {
+static const int DEFAULT_AXIS = 1;
+NodeFusingType GetNodeFusingType(const std::shared_ptr<const Node> &node) {
+    auto &rt = node->get_rt_info();
+    const auto rinfo = rt.find("MayBeFusedInPlugin");
+    if (rinfo == rt.end())
+        return NodeFusingType::NotSet;
+    return rinfo->second.as<NodeFusingType>();
+}
+void SetNodeFusingType(const std::shared_ptr<Node> &node, NodeFusingType nodeType) {
+    auto &rt = node->get_rt_info();
+    rt["MayBeFusedInPlugin"] = nodeType;
+}
+std::vector<NodeFusingType> getContinuableChains(const std::shared_ptr<const Node> &node) {
+    std::vector<NodeFusingType> result;
+    for (const auto& input : node->inputs()) {
+        const auto parent = input.get_source_output().get_node_shared_ptr();
+        const auto snt = GetNodeFusingType(parent);
+        if (snt > NodeFusingType::FusedTerminator) {
+            result.push_back(snt);
+        }
+    }
+    return result;
+}
+int getNumNonConstInputs(const std::shared_ptr<const Node> &node) {
+    int num_non_const_inputs = 0;
+    for (const auto &parent_out : node->input_values()) {
+        const auto parent = parent_out.get_node_shared_ptr();
+        if (ov::is_type<ov::op::v1::Reshape>(parent)) {
+            for (const auto &grandparent_out : parent->input_values()) {
+                const auto grandparent = grandparent_out.get_node_shared_ptr();
+                if (!ov::is_type<ov::op::v0::Constant>(grandparent))
+                    num_non_const_inputs++;
+            }
+        } else if (!ov::is_type<ov::op::v0::Constant>(parent)) {
+            num_non_const_inputs++;
+        }
+    }
+    return num_non_const_inputs;
+}
+bool isFullyConnected(const std::shared_ptr<const ov::Node>& node) {
+    if (!ov::is_type<ov::op::v0::MatMul>(node))
+        return false;
+    const auto out_activations = node->input_value(0);
+    const auto out_weights = node->input_value(1);
+    const auto rank_a = out_activations.get_partial_shape().rank();
+    const auto rank_w = out_weights.get_partial_shape().rank();
+    return out_weights.get_partial_shape().is_static() &&
+           rank_a.is_static() && rank_w.is_static() &&
+           rank_a.get_length() != 1 && rank_w.get_length() != 1 &&
+           rank_w.get_length() <= 3 &&
+           ov::op::util::is_on_constant_path(out_weights);
+}
+
+bool SupportsFusingWithConvolution_Simple(const std::shared_ptr<const Node> &node) {
+    // Remove comment when these nodes will be supported by arm in snippets
+    // return ov::is_type<ov::op::v0::Relu>(node) ||
+    //        ov::is_type<ov::op::v0::Tanh>(node) ||
+    //        ov::is_type<ov::op::v0::Elu>(node) ||
+    //        ov::is_type<ov::op::v0::Abs>(node) ||
+    //        ov::is_type<ov::op::v0::Sqrt>(node) ||
+    //        ov::is_type<ov::op::v4::SoftPlus>(node) ||
+    //        ov::is_type<ov::op::v0::Sigmoid>(node) ||
+    //        ov::is_type<ov::op::v0::Clamp>(node);
```

Comment:
Minor: Please provide the ticket number.

Reply:
In general, it looks a little like a piece of debugging code
If would be better to paraphrase to smth like: 
```suggestion
    Note: some other operations support this fusing: (Sqrt, SoftPlus, ...). Skip them here, when they are supported by Snippets ARM. Ticket: xxxx
```

Reply:
Applied. Thanks Ivan!